features/ai-chat.json

```diff
@@ -3,5 +3,18 @@
         "description": "Browser integration with DuckDuckGo's AI Chat"
     },
     "state": "disabled",
-    "exceptions": []
+    "exceptions": [],
+    "features": {
+        "toolbarShortcut": {
+            "state": "disabled"
+        },
+        "applicationMenuShortcut": {
+            "state": "disabled"
+        }
+    },
+    "settings": {
+        "aiChatURL": "https://duckduckgo.com/?q=DuckDuckGo+AI+Chat&ia=chat&duckai=2",
```

Comment:
Do these settings apply for Windows also? (see the build output). The keys don't deep merge so often they're "better" in the overrides/ file.

Reply:
_> Do these settings apply for Windows also?_ 
* They should, yes, just waiting for the green flag from the window team in the [TD](https://app.asana.com/0/481882893211075/1208540070228419/f).  (this should have been a draft PR)
* That said, since there's no recent plan of adding them to iOS and Android, I'll change this to be an override only on macOS and leave windows out of it for the time being.